### PR TITLE
Enable skyscrapers on Dental sites

### PR DESCRIPTION
**Pull request by Alixers505**
https://www.pivotaltracker.com/story/show/172360304

Added skyscraper ad to all dental sites: home, website-section, content

### CODE REVIEW

File: sites/dentaleconomics.com/server/templates/website-section/index.marko [hunk 1]
```diff
@@ -6,7 +6,7 @@ $ const { id, alias, name, pageNode } = data;
   name=name
   page-node=pageNode
 >
-  <@load-more name="card-deck">
-    <@query-params limit=10 skip=12 />
+  <@load-more >
```
Comment (zarathustra323) [inline sites/dentaleconomics.com/server/templates/website-section/index.marko:9]: There's an extra space at the end of the element.

**Comment by zarathustra323** [comment]
@Alixers505 can you also provide screenshots with these ads loaded?

**Comment by Alixers505** [comment]
Screen Shot 2020-04-17 at 4 38 18 PM
Screen Shot 2020-04-17 at 4 38 09 PM
Screen Shot 2020-04-17 at 4 37 53 PM
Screen Shot 2020-04-17 at 4 37 39 PM

File: sites/dentaleconomics.com/server/templates/website-section/index.marko (hunk 1 above)
Comment (Alixers505) [reply sites/dentaleconomics.com/server/templates/website-section/index.marko:9]: This screenshot looks like it came from the draft PR, before I updated the website section to:

Screen Shot 2020-04-17 at 4 40 29 PM

**Pull request merged**